### ISSUE TITLE
Fix PHP timezone setting

### ISSUE THREAD

**Issue by pperez** [zabbix/zabbix-community-docker#20]
There's a problem setting a PHP timezone (eg: America/Santiago), i think it's because the regex is using _/_ as a separator on [/config/init/bootstrap.sh](https://github.com/zabbix/zabbix-community-docker/blob/master/Dockerfile/zabbix-server-2.4/container-files-zabbix/config/init/bootstrap.sh#L133)

I guess that changing _/_ to _#_ in the regex should do the trick

**Comment by jangaraj** [comment]
Thx. Shame on me ;-) I'll change regexp.

**Comment by jangaraj** [comment]
TODO: Sed pattern delimiter ('/') should be escaped in user input - it can be also problem for other user variables, e.g. passwords, ...

Timezone input preprocessing (escaping /):

```
  # escape sed pattern delimiter
  PHP_date_timezone=$(echo ${PHP_date_timezone} | sed -e 's/\//\\\//g')
  sed -i 's/PHP_date_timezone/'${PHP_date_timezone}'/g' /etc/php.d/zz-zabbix.ini
```

Password input preprocessing (escaping /\&):

```
sed -e 's/\//\\\//g' | sed -e 's/[\&]/\\&/g'
```

^^^ Over engineered solution
KISS: '#' will be used as pattern delimiter.

**Comment by pedropilla** [comment]
PHP_date_timezone="America\/Sao_Paulo" worked for me

**Comment by jangaraj** [comment]
Yes, because you are escaping pattern delimiter for sed. You can use it as a workaround, but it'll be fixed - https://github.com/monitoringartist/zabbix-community-docker/blob/master/Dockerfile/zabbix-2.4/container-files-zabbix/config/init/bootstrap.sh#L133

**Comment by jangaraj** [comment]
Fixed in new Docker image zabbix/zabbix-2.4 (zabbix/zabbix-3.0).
